ecosystem/typescript/sdk/scripts/publish_ans_contract.ts

```diff
@@ -2,8 +2,10 @@ const path = require("path");
 const { execSync } = require("child_process");
 
 const ANS_CORE_FOLDER = "/aptos-names-contracts/core";
-const APTOS_INIT_COMMAND = "aptos init --network local";
-const GET_DEFAULT_PROFILE_COMMAND = "aptos config show-profiles --profile default";
+const APTOS_INVOCATION = process.env.APTOS_INVOCATION || "aptos";
```

Comment:
A comment explaining what this is for would be great.

Reply:
absolutely! I need to make some clean ups before landing it, thanks!